documentation/user_docs/docs/user_guide/Advanced_use.rst

```diff
@@ -379,74 +501,124 @@ To alter the limits along the x, y, or z axes you use the commands ``lx, ly`` an
 
 
 to change the limits along x to be 0 and 2, and so on.
-To change the axes to log-scale, you use the commands ``logx, logy`` and ``logz``, and to change to a linear scale you use ``linx``\ ...etc. A full list of formatting options can be found `here <http://www.libisis.org/User_Manual#Plot_Commands>`__.
+
+To change the axes to log-scale, you use the commands ``logx``, ``logy`` and ``logz``, and to change to a linear scale you
+use ``linx``\ ...etc. A full list of formatting options can be found `here
+<http://www.libisis.org/User_Manual#Plot_Commands>`__.
 
 Fitting
 =======
 
-You can also use Horace to fit your data. It can take quite a long time for the fit to converge, so it is therefore a good idea to provide a good initial guess of the fit parameters. You can work these out simulating and then comparing the result to the data by eye.
+You can also use Horace to fit your data. It can take quite a long time for the fit to converge, so it is therefore a
+good idea to provide a good initial guess of the fit parameters. You can work these out simulating and then comparing
+the result to the data by eye.
 
-For an introduction and overview of how to use the following fitting functions, please read :ref:`Fitting data <manual/Multifit:Multifit>`. For comprehensive help, please use the Matlab documentation for the various fitting functions that can be obtained by using the ``doc`` command, for example ``doc d1d/multifit`` (for fitting function like Gaussians to d1d objects) or ``doc sqw/multifit_sqw`` (fitting models for S(Q,w) to sqw objects).
+For an introduction and overview of how to use the following fitting functions, please read :ref:`Fitting data
+<manual/Multifit:Multifit>`. For comprehensive help, please use the Matlab documentation for the various fitting
+functions that can be obtained by using the ``doc`` command, for example ``doc d1d/multifit`` (for fitting function like
+Gaussians to d1d objects) or ``doc sqw/multifit_sqw`` (fitting models for S(Q,w) to sqw objects).
 
 
 Simulating
 ==========
 
-There are two functions used for doing simulations - ``func_eval`` and ``sqw_eval``. The difference between these two functions is relatively minor, and relates to the format of the function that you wish to simulate.
+There are two functions used for doing simulations - ``func_eval`` and ``sqw_eval``. The difference between these two
+functions is relatively minor, and relates to the format of the function that you wish to simulate.
```

Comment:
It would be very frustrating to read this as a new user, and think - yes, but what are the differences? I agree that the old difference description is opaque and uninformative, but a good one looks needed here.

More generally, what does simulating an object mean? A definition really looks needed here